### PR TITLE
Adding AnyABI loader

### PR DESCRIPTION
**Pull request by k-thornton**
Adding AnyABI to the list of ABI loaders

Resolves https://github.com/shazow/whatsabi/issues/90

### CODE REVIEW

**Comment by k-thornton** [comment]
I ran a couple of tests with loadAbi, and autoLoad with loadContractResult = true and false.  Seemed to work ok.  

For testing, I found a valid contract address that AnyABI wasn't aware of
ChainId: 8453, address: 0x5525a074fcfae303d5fcc1ee6ed6c40f5ff8811a

I tested that it loaded ABIs successfully from Ethereum and Base, but didn't test much further.

**Comment by shazow** [comment]
Thank you, I'll try to get this reviewed soon!

File: src/loaders.ts [hunk 1]
```diff
@@ -602,6 +601,68 @@ export type BlockscoutContractResult = {
     has_methods_write_proxy?: boolean;
 };
 
+
+function isAnyABINotFound(error: any): boolean {
```
Comment (shazow) [inline src/loaders.ts:605]: Starting to think we need an `ABILoader.isNotFound(error:any): boolean` that we can inherit/override, but maybe premature?

**Review by shazow — APPROVED** [review]
Looks good, thank you for doing the leg work on this!!

I'll merge once we have some tests to exercise it (welcome to add some here or I can work on it in another PR), just so it doesn't fall between the cracks. :)

**Comment by shazow** [comment]
Merged via #163

**Pull request closed without merging**